core/backend/src/workspace/Workspace.ts

```diff
@@ -657,10 +657,11 @@ export class ITwinWorkspaceDb implements WorkspaceDb {
 }
 
 /**
- * An editable [[WorkspaceDb]]. This is used by administrators for creating and modifying `WorkspaceDb`s.
+ * An editable [[WorkspaceDb]]. This is used by administrators for creating and modifying `WorkspaceDb`s and should
+ * only be used in a dedicated "administrative settings manager" application - never in a user facing application.
  * For cloud-backed containers, the write token must be obtained before this class may be used. Only one user at at time
  * may be editing.
- * @internal
+ * @beta
```

Comment:
It seems to me that only this class needs to become public, right?

Reply:
extract-api is going to fail. You can't give a derived class a higher release tag than its base class.

Reply:
reworked